### PR TITLE
Update MacOS Repo Setup Instructions

### PR DESCRIPTION
**Pull request by TanayParikh**

### CODE REVIEW

File: .github/DEVELOPMENT.md [hunk 1]
```diff
@@ -32,20 +33,42 @@ Install the .NET MAUI workload using the versions from a particular branch:
...
 .\maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
-``` 
+```
+
+MacOS:
+
+```
+curl -O https://aka.ms/dotnet/maui/maui-install.ps1
+chmod +x maui-install.ps1
+.\maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
```
Comment (TanayParikh) [inline .github/DEVELOPMENT.md:49]: This script is still failing due to a missing sudo on line 116 where we install workloads. Could we potentially add the following to line 116 of the script (not available on github as far as I know).

```
    if ($IsLinux -or $IsMacOS) {
        sudo dotnet workload install @Workloads --from-rollback-file $rollbackFile --configfile $nugetConfigFile
    } else {
        dotnet workload install @Workloads --from-rollback-file $rollbackFile --configfile $nugetConfigFile
    }
```

**Comment by TanayParikh** [comment]
Related: https://github.com/dotnet/maui/pull/5560

**Comment by TanayParikh** [comment]
> Needs rebase

Updated.

File: .github/DEVELOPMENT.md [hunk 2]
```diff
@@ -32,18 +33,34 @@ Install the .NET MAUI workload using the versions from a particular branch:
...
-``` 
+```
+
+MacOS:
+
+```bash
+curl -O -L https://aka.ms/dotnet/maui/maui-install.ps1
+chmod +x maui-install.ps1
+sudo pwsh
+./maui-install.ps1 -b 'release/6.0.2xx-preview14' -v '6.0.200-preview'
```
Comment (jonathanpeppers) [inline .github/DEVELOPMENT.md:50]: What does this look like if you do `sudo pwsh -File`, can you remove the `chmod` in that case?

Maybe we can drop two commands in this example?

Comment (TanayParikh) [reply .github/DEVELOPMENT.md]: Ah nice catch! Updated 😄

**Review by jonathanpeppers — APPROVED** [review]

**Pull request merged**